src/GraphQL/GraphQL.csproj

```
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
     <Description>GraphQL for .NET</Description>
     <Authors>Joe McBride</Authors>
     <Copyright>Copyright 2015-2017 Joseph T. McBride et al. All rights reserved.</Copyright>
     <PackageTags>GraphQL;json;api</PackageTags>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL-Parser" Version="6.0.0" />
+    <PackageReference Include="GraphQL-Parser" Version="7.0.0-preview-8" />
```

@Shane32 I tested this on my local machine. I made this package for myself for debugging. What to do to pick up the real package? I understand that now it does not exist at all.

---

I would just push the PR to develop (in parser). Then we can use the package from here. If we need to revert or something in the parser library, so be it.

---

It won’t be on nuget but we will have access to the version within GitHub Packages

---

OK

---

```suggestion
    <PackageReference Include="GraphQL-Parser" Version="7.0.0-preview-20" />
```
So now this is all that is necessary, and the CI builds can retrieve the package.  You may need to set up your machine to pull from that source however.

---

Already done. Locally I use hand-made package because I don't want to deal with GitHub Packages auth.